### ISSUE TITLE
what is the initial learning rate

### ISSUE THREAD

**Issue by Wenhua-Hu** [jiaweizzhao/GaLore#74]
Hello,
I am reproducing some work using your methods. As mentioned in you paper, you train models with 10K training steps.

The llama-1b has a inital learning rate: 0.01.

During the reproducing the galore with lr 0.01, the first 1K evalation shows high loss than your plotting in paper.
Eval loss at step 1000: 4.74, perplexity: 114.60

Can you confirm if you use LR=0.01 for llama-1b ？  and for the comparison, what the best LR you used for training with adam method ?

Thanks,
Wenhua
